protocol-models/src/main/resources/airbyte_protocol/v0/airbyte_protocol.yaml

```diff
@@ -465,11 +465,13 @@ definitions:
     required:
       - name
```

Comment:
Shouldn't we remove `name` here too? Or what would be the `name` of a destination catalog stream if we also have `destination_object_name`?